### ISSUE TITLE
Instance Terminated

### ISSUE THREAD

**Issue by 21CreativeMedia** [xmrig/xmrig#2797]
Im Using xmrig-6.16.2-linux-static-x64.tar.gz
After Runing ./xmrig.sh the instance instantly terminated
Currently Using Xeon Platinum 8380 ICE LAKE
Problem was completely gone while using xmrig-6.16.2-linux-x64.tar.gz
Any Help?

**Comment by Lonnegan** [comment]
You have an Ice-Lake based CPU mit VAES support? The static version of the xmrig binaries are using the VAES instructions, the dynamic one doesn't. So there must be still something wrong, hence there were fixes regarding VAES in the latest release.

What did you want to mine? RandomX? GhostRider?

**Comment by 21CreativeMedia** [comment]
Yes im trying to mine RTM on FlockPool with GhostRider algo. i've read VAES optimizations not supported by xmrig-6.16.2-linux-x64.tar.gz due old compiler, so im using xmrig-6.16.2-linux-static-x64.tar.gz instead. 
but it failed to run on my system,  

been tried on Ryzen 9 5950X and its seem working fine, so  i think there was a problem with Ice Lake Microarch or smth?

**Comment by Lonnegan** [comment]
> been tried on Ryzen 9 5950X and its seem working fine, so i think there was a problem with Ice Lake Microarch or smth?

Well, the devs of xmrig have a Ryzen 5 5600X as their personal computer, so bugs or errors are immediately obvious. They don't have an Ice-Lake system as far as I know, so it's untested there. Perhaps you can help them to find the glitch in the code in combination with Ice Lake :-)
